### ISSUE TITLE
cannot open ux0:app/AUTOPLUGO/menu.lua in v3.20

### ISSUE THREAD

**Issue by fabiomanzoni** [theheroGAC/Autoplugin#43]
Just installed the latest autoplugin and got this error in my PSVITA v3.68.

**Comment by theheroGAC** [comment]
Yes,sorry :(,download this version https://github.com/theheroGAC/Autoplugin/releases/download/3.20/Autoplugin.vpk
